src/AsyncGenerator.yml

```diff
@@ -1,13 +1,13 @@
-﻿projects:
+projects:
 - filePath: NHibernate/NHibernate.csproj
-  targetFramework: netcoreapp2.0
+  targetFramework: netcoreapp2.1
```

Comment:
This failed with following error:

```
crit: AsyncGenerator.CommandLine[0]
      Unable to find System.Threading.Tasks.Task type
      System.InvalidOperationException: Unable to find System.Threading.Tasks.Task type
         at AsyncGenerator.Internal.ThreadSleepAsyncCounterpartFinder.Initialize(Project project, IProjectConfiguration configuration, Compilation compilation) in C:\Workspace\Git\AsyncGenerator\Source\AsyncGenerator\Internal\ThreadSleepAsyncCounterpartFinder.cs:line 30
         at AsyncGenerator.AsyncCodeGenerator.GenerateProject(ProjectData projectData, ILoggerFactory loggerFactory, ILogger logger, CancellationToken cancellationToken)
         at AsyncGenerator.AsyncCodeGenerator.GenerateAsync(AsyncCodeConfiguration configuration, CancellationToken cancellationToken) in C:\Workspace\Git\AsyncGenerator\Source\AsyncGenerator\AsyncCodeGenerator.cs:line 84
         at AsyncGenerator.CommandLine.Program.Main(String[] args) in C:\Workspace\Git\AsyncGenerator\Source\AsyncGenerator.CommandLine\Program.cs:line 51
```

Reply:
The `targetFramework` can be set only to targets that the project supports, for `NHibernate` project it can be set to one of the following: `net461, netcoreapp2.0, netstandard2.0, netstandard2.1`

Reply:
Yeah, I'm stupid...